lib/mail_room.rb

```diff
@@ -2,16 +2,7 @@
 require 'optparse'
 require 'yaml'
 
-# The MailRoom namespace
 module MailRoom
-  # The MailRoom Delivery namespace holds any defined delivery methods
-  #   including:
-  # 
-  #   * postback (default)
-  #   * letter_opener
-  #   * logger
-  #   * noop
-  module Delivery; end
```

Comment:
Any particular reason for these deletes … other than the code/comments being mostly superfluous?